File: core/src/main/java/com/views/calendar/cosmocalendar/view/customviews/SquareTextView.java
package com.views.calendar.cosmocalendar.view.customviews;

import android.content.Context;

import androidx.appcompat.widget.AppCompatTextView;

//TODO convert kotlin
public class SquareTextView extends AppCompatTextView {

    public SquareTextView(Context context) {
        super(context);
    }

    //Square view
    @Override
    public void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec);
    }
}

File: core/src/main/java/com/views/calendar/cosmocalendar/view/customviews/SquareTextView.kt
package com.views.calendar.cosmocalendar.view.customviews

import android.content.Context
import androidx.appcompat.widget.AppCompatTextView

class SquareTextView(context: Context) : AppCompatTextView(context) {
    // Square view
    public override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
        super.onMeasure(widthMeasureSpec, widthMeasureSpec)
    }
}
